Fix failed TestLoadBalancedFargateManifest_Marshal

```
2019-09-20T01:05:29.6011129Z === RUN   TestLoadBalancedFargateManifest_Marshal
2019-09-20T01:05:29.6011391Z --- FAIL: TestLoadBalancedFargateManifest_Marshal (0.01s)
2019-09-20T01:05:29.6154674Z     lb_fargate_manifest_test.go:84: 
2019-09-20T01:05:29.6155099Z         	Error Trace:	lb_fargate_manifest_test.go:84
2019-09-20T01:05:29.6155336Z         	Error:      	Not equal: 
2019-09-20T01:05:29.6156708Z         	            	expected: "# Your application name will be used in naming your resources\n# like log groups, services, etc.\nname: SweetApp\n# The \"Type\" of the application you're running. For a list of all types that we support see\n# https://github.com/aws/PRIVATE-amazon-ecs-archer/app/template/manifest/\ntype: Load Balanced Web App\n\n# The port exposed through your container. We need to know\n# this so that we can route traffic to it.\ncontainerPort: 80\n\n# Size of CPU\ncpu: 256\n\n# Size of memory\nmemory: 512\n\n# Logging is enabled by default. We'll create a loggroup that is\n# the SweetApp/Stage\nlogging: true\n\n# Determines whether the application will have a public IP or not.\npublic: true\n\n# You can also pass in environment variables as key/value pairs\n#environment-variables:\n#  dog: 'Clyde'\n#  cute: 'hekya'\n#\n# Additional Sidecar apps that can run along side your main application\n#sidecars:\n#  fluentbit:\n#    containerPort: 80\n#    image: 'amazon/aws-for-fluent-bit:1.2.0'\n#    memory: 512\n\n# This section defines each of the release stages\n# and their specific configuration for your app.\nstages:\n  -\n    # The \"environment\" (cluster/vpc/lb) to contain this service.\n    env: test\n    # The number of tasks that we want, at minimum.\n    desiredCount: 1\n    # Any secrets via ARNs\n    #secrets:\n    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd\n  -\n    # The \"environment\" (cluster/vpc/lb) to contain this service.\n    env: prod\n    # The number of tasks that we want, at minimum.\n    desiredCount: 3\n    # Any secrets via ARNs\n    #secrets:\n    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd\n"
2019-09-20T01:05:29.6159090Z         	            	actual  : "# Your application name will be used in naming your resources\r\n# like log groups, services, etc.\r\nname: SweetApp\r\n# The \"Type\" of the application you're running. For a list of all types that we support see\r\n# https://github.com/aws/PRIVATE-amazon-ecs-archer/app/template/manifest/\r\ntype: Load Balanced Web App\r\n\r\n# The port exposed through your container. We need to know\r\n# this so that we can route traffic to it.\r\ncontainerPort: 80\r\n\r\n# Size of CPU\r\ncpu: 256\r\n\r\n# Size of memory\r\nmemory: 512\r\n\r\n# Logging is enabled by default. We'll create a loggroup that is\r\n# the SweetApp/Stage\r\nlogging: true\r\n\r\n# Determines whether the application will have a public IP or not.\r\npublic: true\r\n\r\n# You can also pass in environment variables as key/value pairs\r\n#environment-variables:\r\n#  dog: 'Clyde'\r\n#  cute: 'hekya'\r\n#\r\n# Additional Sidecar apps that can run along side your main application\r\n#sidecars:\r\n#  fluentbit:\r\n#    containerPort: 80\r\n#    image: 'amazon/aws-for-fluent-bit:1.2.0'\r\n#    memory: 512\r\n\r\n# This section defines each of the release stages\r\n# and their specific configuration for your app.\r\nstages:\r\n  -\r\n    # The \"environment\" (cluster/vpc/lb) to contain this service.\r\n    env: test\r\n    # The number of tasks that we want, at minimum.\r\n    desiredCount: 1\r\n    # Any secrets via ARNs\r\n    #secrets:\r\n    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd\r\n  -\r\n    # The \"environment\" (cluster/vpc/lb) to contain this service.\r\n    env: prod\r\n    # The number of tasks that we want, at minimum.\r\n    desiredCount: 3\r\n    # Any secrets via ARNs\r\n    #secrets:\r\n    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd\r\n"
2019-09-20T01:05:29.6160087Z         	            	
2019-09-20T01:05:29.6160229Z         	            	Diff:
2019-09-20T01:05:29.6160381Z         	            	--- Expected
2019-09-20T01:05:29.6160531Z         	            	+++ Actual
2019-09-20T01:05:29.6160687Z         	            	@@ -1,56 +1,56 @@
2019-09-20T01:05:29.6160900Z         	            	-# Your application name will be used in naming your resources
2019-09-20T01:05:29.6161103Z         	            	-# like log groups, services, etc.
2019-09-20T01:05:29.6161263Z         	            	-name: SweetApp
2019-09-20T01:05:29.6161531Z         	            	-# The "Type" of the application you're running. For a list of all types that we support see
2019-09-20T01:05:29.6161708Z         	            	-# https://github.com/aws/PRIVATE-amazon-ecs-archer/app/template/manifest/
2019-09-20T01:05:29.6161886Z         	            	-type: Load Balanced Web App
2019-09-20T01:05:29.6162091Z         	            	+# Your application name will be used in naming your resources
2019-09-20T01:05:29.6162282Z         	            	+# like log groups, services, etc.
2019-09-20T01:05:29.6162432Z         	            	+name: SweetApp
2019-09-20T01:05:29.6162698Z         	            	+# The "Type" of the application you're running. For a list of all types that we support see
2019-09-20T01:05:29.6162878Z         	            	+# https://github.com/aws/PRIVATE-amazon-ecs-archer/app/template/manifest/
2019-09-20T01:05:29.6163041Z         	            	+type: Load Balanced Web App
2019-09-20T01:05:29.6163173Z         	            	+
2019-09-20T01:05:29.6163385Z         	            	+# The port exposed through your container. We need to know
2019-09-20T01:05:29.6163588Z         	            	+# this so that we can route traffic to it.
2019-09-20T01:05:29.6163748Z         	            	+containerPort: 80
2019-09-20T01:05:29.6163862Z         	            	+
2019-09-20T01:05:29.6164010Z         	            	+# Size of CPU
2019-09-20T01:05:29.6164154Z         	            	+cpu: 256
2019-09-20T01:05:29.6164285Z         	            	+
2019-09-20T01:05:29.6164416Z         	            	+# Size of memory
2019-09-20T01:05:29.6164562Z         	            	+memory: 512
2019-09-20T01:05:29.6164692Z         	            	+
2019-09-20T01:05:29.6164920Z         	            	+# Logging is enabled by default. We'll create a loggroup that is
2019-09-20T01:05:29.6165194Z         	            	+# the SweetApp/Stage
2019-09-20T01:05:29.6165343Z         	            	+logging: true
2019-09-20T01:05:29.6165565Z         	            	+
2019-09-20T01:05:29.6165790Z         	            	+# Determines whether the application will have a public IP or not.
2019-09-20T01:05:29.6165948Z         	            	+public: true
2019-09-20T01:05:29.6166061Z         	            	+
2019-09-20T01:05:29.6166280Z         	            	+# You can also pass in environment variables as key/value pairs
2019-09-20T01:05:29.6166516Z         	            	+#environment-variables:
2019-09-20T01:05:29.6166669Z         	            	+#  dog: 'Clyde'
2019-09-20T01:05:29.6166856Z         	            	+#  cute: 'hekya'
2019-09-20T01:05:29.6166972Z         	            	+#
2019-09-20T01:05:29.6167205Z         	            	+# Additional Sidecar apps that can run along side your main application
2019-09-20T01:05:29.6167570Z         	            	+#sidecars:
2019-09-20T01:05:29.6167828Z         	            	+#  fluentbit:
2019-09-20T01:05:29.6168102Z         	            	+#    containerPort: 80
2019-09-20T01:05:29.6168275Z         	            	+#    image: 'amazon/aws-for-fluent-bit:1.2.0'
2019-09-20T01:05:29.6168488Z         	            	+#    memory: 512
2019-09-20T01:05:29.6168618Z         	            	+
2019-09-20T01:05:29.6168817Z         	            	+# This section defines each of the release stages
2019-09-20T01:05:29.6169024Z         	            	+# and their specific configuration for your app.
2019-09-20T01:05:29.6169155Z         	            	+stages:
2019-09-20T01:05:29.6169388Z         	            	+  -
2019-09-20T01:05:29.6169613Z         	            	+    # The "environment" (cluster/vpc/lb) to contain this service.
2019-09-20T01:05:29.6169849Z         	            	+    env: test
2019-09-20T01:05:29.6170036Z         	            	+    # The number of tasks that we want, at minimum.
2019-09-20T01:05:29.6170213Z         	            	+    desiredCount: 1
2019-09-20T01:05:29.6170389Z         	            	+    # Any secrets via ARNs
2019-09-20T01:05:29.6170639Z         	            	+    #secrets:
2019-09-20T01:05:29.6170835Z         	            	+    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd
2019-09-20T01:05:29.6170972Z         	            	+  -
2019-09-20T01:05:29.6171234Z         	            	+    # The "environment" (cluster/vpc/lb) to contain this service.
2019-09-20T01:05:29.6171438Z         	            	+    env: prod
2019-09-20T01:05:29.6171641Z         	            	+    # The number of tasks that we want, at minimum.
2019-09-20T01:05:29.6171808Z         	            	+    desiredCount: 3
2019-09-20T01:05:29.6171958Z         	            	+    # Any secrets via ARNs
2019-09-20T01:05:29.6172119Z         	            	+    #secrets:
2019-09-20T01:05:29.6172322Z         	            	+    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd
2019-09-20T01:05:29.6172480Z         	            	 
2019-09-20T01:05:29.6172693Z         	            	-# The port exposed through your container. We need to know
2019-09-20T01:05:29.6172877Z         	            	-# this so that we can route traffic to it.
2019-09-20T01:05:29.6173039Z         	            	-containerPort: 80
2019-09-20T01:05:29.6173229Z         	            	-
2019-09-20T01:05:29.6173416Z         	            	-# Size of CPU
2019-09-20T01:05:29.6173561Z         	            	-cpu: 256
2019-09-20T01:05:29.6173674Z         	            	-
2019-09-20T01:05:29.6173822Z         	            	-# Size of memory
2019-09-20T01:05:29.6173967Z         	            	-memory: 512
2019-09-20T01:05:29.6174097Z         	            	-
2019-09-20T01:05:29.6174299Z         	            	-# Logging is enabled by default. We'll create a loggroup that is
2019-09-20T01:05:29.6174475Z         	            	-# the SweetApp/Stage
2019-09-20T01:05:29.6174672Z         	            	-logging: true
2019-09-20T01:05:29.6174844Z         	            	-
2019-09-20T01:05:29.6175067Z         	            	-# Determines whether the application will have a public IP or not.
2019-09-20T01:05:29.6175208Z         	            	-public: true
2019-09-20T01:05:29.6175453Z         	            	-
2019-09-20T01:05:29.6175702Z         	            	-# You can also pass in environment variables as key/value pairs
2019-09-20T01:05:29.6175876Z         	            	-#environment-variables:
2019-09-20T01:05:29.6176010Z         	            	-#  dog: 'Clyde'
2019-09-20T01:05:29.6176162Z         	            	-#  cute: 'hekya'
2019-09-20T01:05:29.6176292Z         	            	-#
2019-09-20T01:05:29.6176987Z         	            	-# Additional Sidecar apps that can run along side your main application
2019-09-20T01:05:29.6177429Z         	            	-#sidecars:
2019-09-20T01:05:29.6177592Z         	            	-#  fluentbit:
2019-09-20T01:05:29.6177892Z         	            	-#    containerPort: 80
2019-09-20T01:05:29.6178162Z         	            	-#    image: 'amazon/aws-for-fluent-bit:1.2.0'
2019-09-20T01:05:29.6178322Z         	            	-#    memory: 512
2019-09-20T01:05:29.6178454Z         	            	-
2019-09-20T01:05:29.6178635Z         	            	-# This section defines each of the release stages
2019-09-20T01:05:29.6179151Z         	            	-# and their specific configuration for your app.
2019-09-20T01:05:29.6179652Z         	            	-stages:
2019-09-20T01:05:29.6179795Z         	            	-  -
2019-09-20T01:05:29.6180008Z         	            	-    # The "environment" (cluster/vpc/lb) to contain this service.
2019-09-20T01:05:29.6180178Z         	            	-    env: test
2019-09-20T01:05:29.6180449Z         	            	-    # The number of tasks that we want, at minimum.
2019-09-20T01:05:29.6180620Z         	            	-    desiredCount: 1
2019-09-20T01:05:29.6180817Z         	            	-    # Any secrets via ARNs
2019-09-20T01:05:29.6180952Z         	            	-    #secrets:
2019-09-20T01:05:29.6181163Z         	            	-    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd
2019-09-20T01:05:29.6181319Z         	            	-  -
2019-09-20T01:05:29.6181543Z         	            	-    # The "environment" (cluster/vpc/lb) to contain this service.
2019-09-20T01:05:29.6181701Z         	            	-    env: prod
2019-09-20T01:05:29.6181890Z         	            	-    # The number of tasks that we want, at minimum.
2019-09-20T01:05:29.6182055Z         	            	-    desiredCount: 3
2019-09-20T01:05:29.6182250Z         	            	-    # Any secrets via ARNs
2019-09-20T01:05:29.6182403Z         	            	-    #secrets:
2019-09-20T01:05:29.6182601Z         	            	-    #  lemonaidpassword: arn:aws:secretsmanager:us-west-2:902697171733:secret:DavidsLemons/DavidsFrontEnd
2019-09-20T01:05:29.6182742Z         	            	-
2019-09-20T01:05:29.6183736Z         	Test:       	TestLoadBalancedFargateManifest_Marshal
```

--- a/internal/pkg/manifest/lb_fargate_manifest_test.go
+++ b/internal/pkg/manifest/lb_fargate_manifest_test.go
@@ -4,6 +4,7 @@
 package manifest
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
@@ -81,5 +82,5 @@ stages:
 
 	// THEN
 	require.NoError(t, err)
-	require.Equal(t, wantedContent, string(b))
+	require.Equal(t, wantedContent, strings.Replace(string(b), "\r\n", "\n", -1))
 }

